src/org/mozilla/javascript/ScriptableObject.java

```diff
@@ -253,8 +253,17 @@ ScriptableObject getPropertyDescriptor(Context cx, Scriptable scope) {
             if (getter == null && setter == null) {
                 desc.defineProperty("writable", (attr & READONLY) == 0, EMPTY);
             }
-            if (getter != null) desc.defineProperty("get", getter, EMPTY);
-            if (setter != null) desc.defineProperty("set", setter, EMPTY);
+            if (getter != null)
+			{
+				if( getter instanceof MemberBox ) desc.defineProperty("get", new FunctionObject("f", ((MemberBox)getter).member(),scope), EMPTY);
```

Comment:
Sadly there is not a single code style for this huge, old code base, but generally we follow the style of the code "around us" when making a change. You can see that this change is pretty different from other Rhino code. Specifically:

1) Please use spaces and not tabs
2) Put the spaces on the other side of the parentheses (like you see elsewhere in the file)
3) It's not a hard and fast rule yet, but using brackets for every "if" and "else" statement, even if there is only a single statement, avoids lots of errors later.

(I also think this is why the Travis CI build is failing for this PR.)